### ISSUE TITLE
Did Curses.com change their site layout? Script not working any more :-(

### ISSUE THREAD

**Issue by calvinII** [EfficiencyVI/adcmmp#8]
When I start the script I get the only choice for Modpacks: "99999 other"
This wasn't like this some days ago....

Cheers

**Comment by EfficiencyVI** [comment]
Yeah, it seems like curse changed their site entirely … and not in a good way. IDK exactly when I can fix this but I'll try.
